client/client.go

```diff
@@ -140,7 +140,7 @@ func NewTarget(targetName string, targetPath string) (*Target, error) {
 		return nil, err
 	}
 
-	meta, err := data.NewFileMeta(bytes.NewBuffer(b))
+	meta, err := data.NewFileMeta(bytes.NewBuffer(b), "sha256", "sha512")
```

Comment:
Rather than defining in const.go, if we want to define a slice of supported checksums, it would be good to put it right above the `NewFileMeta` definition just so it's obvious that it's available for use with that function.